lib/app.jsx

```diff
@@ -473,7 +503,10 @@ export const App = connect( mapStateToProps, mapDispatchToProps )( React.createC
 					</ContextMenu>
 				}
 				{ state.authorized ?
-						<div className={mainClasses}>
+						<div className={mainClasses}
+							onMouseMove={ makeResizable( { isResizing, sourceListSelector: '.source-list', onResizable: this.props.setSourceListCanResize, onUpdateWidth: this.props.setSourceListWidth } ) }
+							onMouseDown={ startResizing( { canResize, onStartResizing: this.props.setSourceListResizing } ) }
+							onMouseUp={ stopResizing( { isResizing, onStopResizing: () => this.props.setSourceListResizing( false ) } ) } >
```

Comment:
Looks like a great opportunity for a reactive observable, which could potentially allow us to remove `isResizing` from our store. I'm a bit confused as to why we need `isResizing` in the first place, as it doesn't appear to impact styling at all. Is this only to detect the mouse movements? If so, maybe it should remain contained here in this component.

```js
const Draggable = React.createClass( {
	componentWillMount() {
		this.dragging = false;
	},

	startDragging( e ) {
		e.preventDefault();

		this.dragging = true;
	},

	stopDragging( e ) {
		e.preventDefault();

		this.dragging = false;
	},

	render() {
		const { children, onDrag } = this.props;

		return React.createElement( children, {
			onMouseDown: this.startDragging,
			onMouseUp: this.stopDragging,
			onMouseMove: onDrag
		} ;
	}
} );

<Draggable onDrag={ e => onUpdateWidth( e.clientX ) }>
	<div className={ mainClasses }>
		<NavigationBar …>
		…
	</div>
</Draggable>
```